dpcpp/base/executor.dp.cpp

```diff
@@ -150,6 +159,32 @@ int DpcppExecutor::get_num_devices(std::string device_type)
 }
 
 
+bool DpcppExecutor::verify_memory_to(const OmpExecutor *dest_exec) const
+{
+    auto device = detail::get_devices(device_type_)[device_id_];
+
+    return device.is_host() || device.is_cpu();
```

Comment:
nit: usually, you did not have an empty line between `device` and `return`:
```suggestion
    return device.is_host() || device.is_cpu();
```